doc/src/fix_graphics.rst

```diff
@@ -0,0 +1,174 @@
+         tics = number of tics (unitless)
+         only the progress ratio value can be a variable (see below)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   fix 1 all graphics 100 sphere 1 0.0 0.0 15.0 3.0 sphere 2 0.0 0.0 5.0 1.0
+   fix 1 all graphics 1000 sphere 1 v_x v_y 0.0 v_radius cylinder 1 v_x v_y 0.0 v_x v_y 10.0 3.0
+   fix 2 all graphics 100 progbar 3 1 z 0.012 -0.012 0.0025 0.03 0.0003 v_prog 10
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+This fix allows to add arbitrary objects to images rendered with
+:doc:`dump image <dump_image>` using the *fix* keyword.
+
+The *Nevery* keyword determines how often the graphics object data is
+updated.  This should be the same value as the corresponding *N*
+parameter of the :doc:`dump <dump>` image command.  LAMMPS will stop
+with an error message if the settings for this fix and the dump command
+are not compatible.
+
+The *type* quantity determines the color of the object.  Its represents
+an *atom* type and the object will be colored the same as the
+corresponding atom type when the *type* coloring scheme is used in the
+:doc:`dump image fix <dump_image>` command is used.  The color may also
+be that of the atom type's element or just a globally set constant color
+for *all* objects of this fix instance, which can be changed using a
+:doc:`dump modify fcolor <dump_image>` command.  For the *progbar*
+object *two* atom type values must be specified.
+
+Available graphics objects are (see above for exact command line syntax):
+
+- *sphere* - a sphere defined by its center location and its radius
+- *cylinder* - a cylinder defined by its two center endpoints and its radius
+- *arrow* - a cylinder with a cone at one side
+- *progbar* - progress bar a long a selected axis and with optional tick marks
+
+Many of the quantities defining a graphics object can be specified as an
+equal-style :doc:`variable <variable>`, namely *x*, *y*, *z*, or *R* for
+a *sphere* or namely *x1*, *y1*, *z1*, *x2*, *y2*, *z2*, or *R* for a
+*cylinder*.  If any of these values is a variable, it should be
+specified as `v_name`, where `name` is the variable name.  In this case,
+the variable will be evaluated each *Nevery* timestep, and its value
+used to define the indenter geometry.
+
+Note that equal-style variables can specify formulas with various
+mathematical functions, and include :doc:`thermo_style <thermo_style>`
+command keywords for the simulation box parameters and timestep and
+elapsed time.  Thus it is easy to specify graphics object properties
+like position, orientation, radius or more that change as a function of
+time or span consecutive runs in a continuous fashion.  For the latter,
+see the *start* and *stop* keywords of the :doc:`run <run>` command and
+the *elaplong* keyword of :doc:`thermo_style custom <thermo_style>` for
+details.
+
+For example, if a sphere's x-position is specified as v_x, then this
+variable definition will keep it's center at a relative position in the
```

Comment:
its center

Reply:
fixed. the text was copied with small adjustments from fix indent, so it fixed it there as well.